saleor/graphql/page/schema.py

```diff
@@ -49,6 +49,12 @@ class PageQueries(graphene.ObjectType):
         PageCountableConnection,
         sort_by=PageSortingInput(description="Sort pages."),
         filter=PageFilterInput(description="Filtering options for pages."),
+        search=graphene.String(
+            description="Search pages. Overrides filter.search input." + ADDED_IN_322
```

Comment:
I guess it's a general question, is `filter.search` (and `filter` in general) deprecated since `where` is available?